hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -400,6 +401,20 @@ public void testTableName() {
     }
   }
 
+  @Test
+  public void testClientPoolCleaner() throws InterruptedException {
+    HiveCatalog catalog = new HiveCatalog(hiveConf);
+    HiveClientPool clientPool1 = catalog.loadHiveClientPool(hiveConf);
+    Thread.sleep(TimeUnit.SECONDS.toMillis(5));
```

Comment:
I have found that `Clock` is very useful when we run tests against current time and do not want to use Thread.sleeps...

Reply:
I choose Thread.sleep() because we are using this pattern in other test suits as well. Anyway, I changed this part to TimeUnit.SECONDS.sleep() for better readability.